src/Render.cc

```diff
@@ -57,13 +64,26 @@ namespace drafter {
             case JSONRenderFormat:
             {
                 refract::RenderJSONVisitor renderer;
-                refract::IElement* element = MSONToRefract(attributes);
+
+                refract::IElement* element = MSONToRefract(*attributes);
+
+                if (!element) {
+                    return body;
+                }
+
```

Comment:
Yes, that is checked by `ExpandRefract` below and transferred the same NULL to expanded and which gets caught by the `if (!expanded)` condition. We need to have the above condition.